apiserver/facades/controller/caasapplicationprovisioner/state.go

```diff
@@ -58,7 +58,7 @@ type Application interface {
 	AllUnits() ([]Unit, error)
 	GetUnitAttachmentInfos() ([]state.UnitAttachmentInfo, error)
 	UpdateUnits(unitsOp *state.UpdateUnitsOperation) error
-	StorageConstraints() (map[string]state.StorageConstraints, error)
+	StorageDirectives() (map[string]state.StorageConstraints, error)
```

Comment:
Am wondering if we should leave this as it was (same with WatchStorageDirectives)
The state stuff deals with StorageConstraints. I think consistency in naming across the state layer is best.

Reply:
sure i'll refactor all the state related interfaces as well